elements/bulbs-video/components/cover.js

```diff
@@ -5,7 +5,6 @@ import VideoMetaRoot from '../elements/meta/components/root';
 
 export default function Cover (props) {
   let { video, actions, enablePosterMeta, disableMetaLink } = props;
-  let imageId = parseInt(video.poster_url.match(/\d+/)[0], 10);
```

Comment:
Per @collin, this property is no longer used and was throwing errors during tests.